metsim/constants.py

```diff
@@ -25,6 +25,7 @@
 SEC_PER_DAY = 86400
 MIN_PER_HOUR = 60
 HOURS_PER_DAY = 24
+MIN_PER_DAY = MIN_PER_HOUR * HOURS_PER_DAY
```

Comment:
If you are calculating `MIN_PER_DAY` this way, then you might as well extend the same treatment to `SEC_PER_DAY`